docs/2-apis/1-swap-api.md

```diff
@@ -314,17 +320,19 @@ const {
   addressLookupTableAddresses, // The lookup table addresses that you can use if you are using versioned transaction.
 } = instructions;
 
-const swapInstruction = new TransactionInstruction({
-  programId: new PublicKey(swapInstructionPayload.programId),
-  keys: swapInstructionPayload.accounts.map((key) => ({
-    pubkey: new PublicKey(key.pubkey),
+const deserializeInstruction = (instruction) => {
+  return new solanaWeb3.TransactionInstruction({
+    programId: new solanaWeb3.PublicKey(instruction.programId),
+    keys: instruction.accounts.map((key) => ({
+      pubkey: new solanaWeb3.PublicKey(key.pubkey),
```

Comment:
can we just use `PublicKey` here?

Reply:
same for everywhere else.

Reply:
oh, sorry, just pasted from my project directly, ofc